oc-aliases.sh

```diff
@@ -2,75 +2,82 @@
 }
 
 # UI Frontend log
-alias uiflog='podlog spectrum-discover-ui-frontend'
+alias uiflog='podlog role=ui,sub-role=frontend'
 # UI Backend log
-alias uiblog='podlog spectrum-discover-ui-backend'
+alias uiblog='podlog role=ui,sub-role=backend'
 # Shell into the running db2wh-rest container
-alias db2ex='podbash spectrum-discover-db2whrest'
+alias db2ex='podbash role=db2whrest'
 # Shell into the running policy engine container
-alias peex='podbash spectrum-discover-policyengine'
+alias peex='podbash role=policyengine'
 # Shell into the running connection management container
-alias conex='podbash spectrum-discover-connmgr'
+alias conex='podbash role=connmgr'
 # Delete db2wh-rest pod
-alias redb2='delpod spectrum-discover-db2whrest'
+alias redb2='delpod role=db2whrest'
 # Db2wh-rest log
-alias db2log='podlog spectrum-discover-db2whrest'
+alias db2log='podlog role=db2whrest'
 # Connection management log
-alias conlog='podlog spectrum-discover-connmgr'
+alias conlog='podlog role=connmgr'
 # Policy Engine log
-alias pollog='podlog spectrum-discover-policyengine'
-alias pelog='podlog spectrum-discover-policyengine'
+alias pollog='podlog role=policyengine'
+alias pelog='podlog role=policyengine'
 # SD Monitor log
-alias sdmonlog='podlog spectrum-discover-sdmonitor'
+alias sdmonlog='podlog role=sdmonitor'
 # Db2wh password
-alias pw='oc exec -it -n=spectrum-discover $(oc get pods -n=spectrum-discover -l app=spectrum-discover-connmgr -o=jsonpath={.items[0].metadata.name}) env |grep DB2WHREST_PASSWORD | cut -d "=" -f 2'
+alias pw='oc -n ${PROJECT_NAME} exec -c spectrum-discover $(oc -n ${PROJECT_NAME} get pods -l role=connmgr -o name) -- env | grep DB2WHREST_PASSWORD | cut -d "=" -f 2'
 # List policies
-alias getpols='tcurl https://${SD_HOST:?environment variable must be set}/policyengine/v1/policies |jq'
+alias getpols='tcurl https://${SD_CONSOLE_ROUTE:?environment variable must be set}/policyengine/v1/policies | jq'
 # List data source connections
-alias getconns='tcurl https://${SD_HOST:?environment variable must be set}/connmgr/v1/connections |jq'
+alias getconns='tcurl https://${SD_CONSOLE_ROUTE:?environment variable must be set}/connmgr/v1/connections | jq'
 # Get/restart scrips for scale/file producers
-alias psplog='podlog spectrum-discover-producer-scale-scan'
-alias repsp='delpod spectrum-discover-producer-scale-scan'
-alias pfplog='podlog spectrum-discover-producer-file-scan'
-alias repfp='delpod spectrum-discover-producer-file-scan'
+alias psplog='podlog role=scale-scan'
+alias repsp='delpod role=scale-scan'
+alias pfplog='podlog role=file-scan'
+alias repfp='delpod role=file-scan'
 # Get TLS certificates
-alias tls='gettoken; curl -k -H "Authorization: Bearer ${TOKEN}"  https://${SD_HOST:?environment variable must be set}/policyengine/v1/tlscert'
+alias tls='gettoken; curl -k -H "Authorization: Bearer ${TOKEN}" https://${SD_CONSOLE_ROUTE:?environment variable must be set}/policyengine/v1/tlscert'
```

Comment:
Once defined SD_CONSOLE_ROUTE, you can remove the ":?environment variable must be set"